demos/social_distance_demo/README.md

```diff
@@ -0,0 +1,147 @@
+    -nstreams "<integer>"      Optional. Number of streams to use for inference on the CPU or/and GPU in throughput mode (for HETERO and MULTI device cases use format <device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>)
+    -nthreads "<integer>"      Optional. Number of threads to use for inference on the CPU (including HETERO and MULTI cases).
+    -u                         Optional. List of monitors to show initially.
+```
+
+Running the application with an empty list of options yields an error message.
+
+To run the demo, you can use public or pre-trained models. To download the pre-trained models, use the OpenVINO [Model Downloader](../../tools/downloader/README.md) or go to [https://download.01.org/opencv/](https://download.01.org/opencv/).
+
+> **NOTE**: Before running the demo with a trained model, make sure the model is converted to the Inference Engine format (\*.xml + \*.bin) using the [Model Optimizer tool](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html).
+
+For example, to do inference on a GPU with the OpenVINO toolkit pre-trained models, run the following command:
+
+```sh
+./social_distance_demo -i <path_to_video>/inputVideo.mp4 -m <path_to_model>/person-detection-retail-0013.xml -m_reid <path_to_model>/person-reidentification-retail-0300.xml -d GPU
+```
+
+To do inference for two video inputs using two asynchronous infer request on FPGA with the OpenVINO toolkit pre-trained models, run the following command:
+```sh
+./social_distance_demo -i <path_to_video>/inputVideo_0.mp4 <path_to_video>/inputVideo_1.mp4 -m <path_to_model>/person-detection-retail-0013.xml -m_reid <path_to_model>/person-reidentification-retail-0300.xml -d HETERO:FPGA,CPU -d_reid HETERO:FPGA,CPU -nireq 2
+```
+
+To do inference for video inputs on Intel® Vision Accelerator Design with Intel® Movidius™ VPUs, some optimization hints are suggested to make good use of the computation ability:
+
+* configuring the number of allocated frames (`-n_iqs`) to provide enough inputs for inference;
+* configuring the number of infer request (`-nireq`) to achieve asynchronous inference;
+* configuring the number of threads (`-n_wt`) for multi-threaded processing.
+
+For example, to run the sample on one Intel® Vision Accelerator Design with Intel® Movidius™ VPUs Compact R card, run the following command:
+```sh
+./social_distance_demo -i <path_to_video>/inputVideo.mp4 -m <path_to_model>/person-detection-retail-0013.xml -m_reid <path_to_model>/person-reidentification-retail-0300.xml  -d HDDL -d_reid HDDL -n_iqs 10 -n_wt 4 -nireq 10
+```
+
+> **NOTE**: For the `-tag` option (HDDL plugin only), you must specify the number of VPUs for each network in the `hddl_service.config` file located in the `<INSTALL_DIR>/deployment_tools/inference_engine/external/hddl/config/` directory using the following tags:
```

Comment:
Not quite relevant to this PR, but it should be noted, that this might be inconvenient. OpenVINO <INSTALL_DIR> might be at location with restricted rights to modify for current user, like "C:\Program Files (x86)" on Windows or "/opt/intel" on Linux. Is there way to provide path to user customized local config file?